### PR TITLE
refactor: version 2

### LINKED ISSUE #69: Update documentation grammars and wordings
- [ ] https://github.com/kodingdotninja/chakra-icons/pull/429

### PR DESCRIPTION
**Pull request by grikomsn**
Closes #69 #70 #71

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,79 +1,44 @@
 {
   "name": "@kodingdotninja/chakra-icons",
   "version": "0.0.0",
-  "workspaces": [
```
Comment (r17x) [inline package.json:4]: This is will be impacted with multi-semantic-release. But I want to find another solutions for handle release.

**Comment by r17x** [comment]
I will try this change in my local. Thanks @grikomsn

File: package.json (hunk 1 above)
Comment (grikomsn) [reply package.json:4]: We could restore the workspaces field

**Comment by r17x** [comment]
Ship it @grikomsn

File: .gitignore [hunk 2]
```diff
@@ -1,59 +1,14 @@
...
-# Directories potentially created on remote AFP share
-.AppleDB
-.AppleDesktop
-Network Trash Folder
-Temporary Items
-.apdisk
-
 tooling/cli/dist/index.bin.js
+
+!**/.gitkeep
```
Comment (r17x) [inline .gitignore:14]: one more, could you add this ? @grikomsn 

```
.direnv
.envrc
```

**Review by r17x — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (grikomsn) [reply .gitignore:16]: c766156fd928aea5b871ca643d5b087e0accda68

**Review by r17x — APPROVED** [review]
Okay, ship it

**Pull request merged**